doc/tutorials/rt_linux.rst

```diff
@@ -44,71 +44,75 @@ system on Intel KBL NUC with a SATA SSD as ``/dev/sda`` and an NVME SSD as
 1. Follow the :ref:`set-up-CL` instructions in the
    :ref:`getting-started-apl-nuc` to:
 
-   a. Install Clear Linux (version 26800 or higher) onto the NVMe
-   #. Install Clear Linux (version 26800 or higher) onto the SATA SSD
+   a. Install Clear Linux (version 29400 or higher) onto the NVMe
+   #. Install Clear Linux (version 29400 or higher) onto the SATA SSD
    #. Set up Clear Linux on the SATA SSD as the Service OS (SOS) following
       the :ref:`add-acrn-to-efi` instructions in the same guide.
 
-#. Patch and build the Real-Time kernel
-
-   a. Download Linux kernel real-time patch::
-
-         $ wget https://mirrors.edge.kernel.org/pub/linux/kernel/projects/rt/4.19/patch-4.19.31-rt18.patch.xz
-
-   #. Sync the kernel code to acrn-2019w17.4-160000p::
-
-         $ git clone https://github.com/projectacrn/acrn-kernel.git
-         $ git checkout acrn-2019w17.4-160000p
-         $ cd acrn-kernel
-         $ wget https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/tutorials/rt_linux.patch 
-         $ git apply rt_linux.patch 
-         $ xzcat ../patch-4.19.31-rt18.patch.xz | patch -p1
-
-   #. Edit the ``kernel_config_uos`` config file: search for the keyword
-      "NVME Support", delete ``# CONFIG_BLK_DEV_NVME is not set`` and add two lines under "NVME Support" to enable
-      the NVME driver in RT kernel::
-
-         CONFIG_NVME_CORE=y
-         CONFIG_BLK_DEV_NVME=y
-
-   #. Build the RT kernel::
-
-         $ cp kernel_config_uos .config
-         $ make targz-pkg
-
-      Choose "Fully Preemptible Kernel (RT)" when prompted, and
-      choose default for all the other options.
-
-   #. Copy the generated package to SOS::
-
-         $ scp linux-4.19.28-rt18-quilt-2e5dc0ac-dirty-x86.tar.gz <user name>@<SOS ip>:~/
-
-#. Configure the system on SOS
-
-
-   a. Extract kernel boot and lib modules from the package::
-
-         $ cd ~/
-         $ tar xzvf linux-4.19.28-rt18-quilt-2e5dc0ac-dirty-x86.tar.gz
-
-   #. Copy the extracted lib modules to NVME SSD::
-
-         $ mount /dev/nvme0n1p3 /mnt
-         $ cp -r ~/lib/modules/4.19.28-rt18-quilt-2e5dc0ac-dirty /mnt/lib/modules
-
-   #. Edit and run the ``launch_hard_rt_vm.sh`` script to launch the UOS.
-      A sample ``launch_hard_rt_vm.sh`` is included in the Clear Linux
-      release, and is also available in the acrn-hypervisor/devicemodel
-      GitHub repo (in the samples folder).
-
-      You'll need to modify two places:
-
-      1. Replace ``/root/rt_uos_kernel`` with ``~/boot/vmlinuz-4.19.28-rt18-quilt-2e5dc0ac-dirty``
-      #. Replace ``root=/dev/sda3`` with ``root=/dev/nvme0n1p3``
-
-   #. Run the launch script::
-
-         $ sudo ./launch_hard_rt_vm.sh
+#. Set up and launch RT-LaaG
```

Comment:
I would suggest the following text (to avoid this acronym that is not explained):
```suggestion
#. Set up and launch a Real-Time Linux guest
```